msamp/deepspeed/runtime/engine.py

```diff
@@ -268,8 +172,9 @@ def _configure_zero_optimizer(self, optimizer, use_fp8=False):
         Returns:
             ZeROOptimizer: zero optimizer.
         """
-        if use_fp8:
-            raise NotImplementedError('fp8 ZeRO is not supported yet')
+        if self.msamp_enabled():
+            raise NotImplementedError('Not implemented on MSAMP')
```

Comment:
Enable ZeRO with MS-AMP after the PR (https://github.com/Azure/MS-AMP/pull/73) is merged.

Reply:
Rebased.